packages/cli/src/commands/link/linkAssets.ts

```diff
@@ -1,15 +1,12 @@
-// @flow
-
 import {isEmpty} from 'lodash';
-import type {PlatformsT, ProjectConfigT} from 'types';
-
+import {Config} from '@react-native-community/cli-types';
 import {logger} from '@react-native-community/cli-tools';
 
 const linkAssets = (
-  platforms: PlatformsT,
-  project: ProjectConfigT,
+  platforms: Config['platforms'],
+  project: Config['project'],
   assets: Array<string>,
-) => {
+): void | null => {
```

Comment:
Hrm, I don't see the `null` return. can this just be void?

Reply:
https://github.com/react-native-community/cli/blob/master/packages/cli/src/commands/link/linkAssets.js#L14 
In this line its `return` thats it....so will it be covered with `void`  ?

Reply:
When a function returns nothing (either by specifcyting `return;` or, well, not specifying it), it implicitly returns `undefined`. `void` in a type def is equivalent to `undefined`, but it's also an expression in JS, that always returns undefined: `void 1 === undefined`, `void 'text' === undefined`.

Reply:
changed to void alone